src/data/calculate.ts

```diff
@@ -82,7 +82,7 @@ export const defaultValues: CalculatorData = {
   yourVaccineType: '',
   yourVaccineDoses: 0,
 
-  theirVaccine: 'undefined',
```

Comment:
I was getting a 'Cannot read property 'value' of undefined' js error from SegmentedControl: https://github.com/microcovid/microcovid/blob/main/src/components/calculator/controls/SegmentedControl.tsx#L57